src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MemberDefHandler.java

```diff
@@ -51,13 +53,38 @@ public void checkIndentation() {
             checkType();
         }
         final DetailAST firstNode = getMainAst();
-        final DetailAST lastNode = getVarDefStatementSemicolon(firstNode);
+        DetailAST lastNode = getVarDefStatementSemicolon(firstNode);
 
-        if (lastNode != null && !isArrayDeclaration(firstNode)) {
+        if (lastNode != null) {
+            lastNode = getArrayInitNode(firstNode).orElse(lastNode);
             checkWrappingIndentation(firstNode, lastNode);
         }
     }
 
+    /**
+     * Finds the array init node.
+     *
+     * @param firstNode Node to begin searching
+     * @return array init node
+     */
+    private static Optional<DetailAST> getArrayInitNode(DetailAST firstNode) {
```

Comment:
I noticed nothing is checking lines and what starts the line, etc...

Shouldn't there be some check to first identify if `=` starts the line or not?
Why do we need to look at the `expr` and `array_init` if `=` is starting the line?

Reply:
The problem was that the line starting with `=` was skipped because `lastNode != null && !isArrayDeclaration(firstNode)` was false. With my changes `checkWrappingIndentation` is called and checks line wraps.